### ISSUE TITLE
Quick Start instructions results in an error

### ISSUE THREAD

**Issue by aidenmitchell** [instructure/canvas-lms#1775]
### Summary:
When following the [Quick Start](https://github.com/instructure/canvas-lms/wiki/Quick-Start) instructions, an error is thrown.
```
> Copying default configuration from config/docker-compose.override.yml.example to docker-compose.override.yml

> Now we can set up Canvas!

> Copying Canvas docker configuration...
OK to run 'cp docker-compose/config/*.yml config/'? [y/n] y

> Building docker images...
ERROR: Need service name for --build-arg option
```
### Steps to reproduce:

 1. Follow Quick Start instructions
 2. Get to `OK to run 'cp docker-compose/config/*.yml config/'? [y/n]`

### Expected behavior:
Quick Start continues and completes successfully.
### Actual behavior:
An error is thrown: `ERROR: Need service name for --build-arg option`
### Additional notes:
Running on Ubuntu 18.04

**Comment by raheems** [comment]
Same here. Any fix?

**Comment by aidenmitchell** [comment]
Nope, still waiting unfortuantly.

**Comment by lightbord** [comment]
FWIW, I've solved this on Ubuntu Server 18.04.4 by `sudo apt-get remove docker-compose` (**DO NOT** `autoremove` which would remove dependencies as well) and installing the latest version via the instructions here: [https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/). The command to download the current version as of this writing (1.27.4) is `sudo curl -L "https://github.com/docker/compose/releases/download/1.27.4/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose`. Then, `sudo chmod +x /usr/local/bin/docker-compose`.

The issue seems to be related to the version of `docker-compose` Ubuntu installs, which was 1.17.1, far outdated from the current version of 1.27.4.

**Comment by jbutte** [comment]
lightbord is correct, it appears to be an issue with the version of docker-compose. We've modified docker_dev_setup.sh to now check for dependencies rather than trying to install them. Now when you run docker_dev_setup.sh, if you are missing a dependency you will be notified and given some help with how to install them. We've included the docker-compose specifics as mentioned in lightbord's comment. This is the commit with those changes: https://github.com/instructure/canvas-lms/commit/97fb3053efe22f8e1eb83777abefc43321c0611d
